pkg/reconciler/route/route.go

```diff
@@ -369,7 +369,7 @@ func (c *Reconciler) clusterLocalDomainTLS(ctx context.Context, r *v1.Route, tc
 
 	orphanCerts, err := c.getOrphanRouteCerts(r, usedDomains, netcfg.CertificateClusterLocalDomain)
 	if err != nil {
-		return nil, nil
+		return nil, nil //nolint:nilerr
```

Comment:
Can you undo this `nolint` directive. I left this unfixed in an earlier PR so I could double check why we're not handling this scenario

Reply:
We can do an override to get passed this lint failure for this PR and deal with it in a follow up